robosuite/devices/spacemouse.py

```diff
@@ -128,7 +129,12 @@ def __init__(
         self.vendor_id = vendor_id
         self.product_id = product_id
         self.device = hid.device()
-        self.device.open(self.vendor_id, self.product_id)  # SpaceMouse
+        try:
+            self.device.open(self.vendor_id, self.product_id)  # SpaceMouse
+        except OSError as e:
+            print("Failed to open SpaceMouse device: ", e)
```

Comment:
Could we please use `ROBOSUITE_DEFAULT_LOGGER.warning` instead of print statements?

Reply:
for sure, the change has been made and the termcolor import has been removed

Reply:
thanks!